cypress.config.ts

```diff
@@ -39,7 +40,18 @@ export default defineConfig({
         ...frontendComponentApi,
         ...plpUi,
       })
+      on('after:spec', (spec: Cypress.Spec, results: CypressCommandLine.RunResult) => {
```

Comment:
Thank you 👍 I was going to sneak this into one of my CIAG PRs as well, so you've saved me a job 😁